src/components/v2/V2Create/tabs/RulesTab/RulesTabContent.tsx

```diff
@@ -37,22 +43,21 @@ export default function RulesTabContent() {
       dispatch(editingV2ProjectActions.setPausePay(fields.pausePay))
       dispatch(editingV2ProjectActions.setPauseMint(fields.pauseMint))
       dispatch(editingV2ProjectActions.setBallot(fields.ballot))
+      onFinish?.()
     },
-    [dispatch],
+    [dispatch, onFinish],
   )
 
   const resetForm = useCallback(() => {
     form.setFieldsValue({
-      pausePay: fundingCycleMetadata?.pausePay ?? false,
-      pauseMint: fundingCycleMetadata?.pauseMint ?? false,
+      pausePay: fundingCycleMetadata?.pausePay,
```

Comment:
**question** why this change? does `undefined` produce the same outcome? feel like we could be explicit and keep it as `false`

Reply:
`undefined` does produce the same outcome